tree/tree/Extension/Extension+UITableViewDelegate.swift

```diff
@@ -0,0 +1,98 @@
+        _ imageData: Data?,
+        _ articleIdentifier: String?,
+        _ articleType: ScrappedArticleType?,
+        completion: @escaping (Bool) -> Void
+        ) -> UIContextualAction {
+        switch type {
+        case .scrap:
+            guard let articleData = articleData else { return UIContextualAction() }
+            var isScrapped: Bool = true
+            var scrppaedArticle: ScrappedArticle?
+            ScrapManager.articleIsScrapped(uri: articleData.uri) { (articleScrapped, article) in
+                isScrapped = articleScrapped
+                scrppaedArticle = article
+            }
+            let scrapAction = UIContextualAction(
+                style: .normal,
+                title: nil) { (_, _, success) in
+                    if !isScrapped {
+                        ScrapManager.scrapArticle(
+                            .search ,
+                            articleStruct:
+                            SearchedArticleStruct(articleData: articleData, imageData: imageData)
+                        )
+                    } else {
+                        if let tempscrppaedArticle = scrppaedArticle {
+                            ScrapManager.removeArticle(tempscrppaedArticle)
+                        }
+                    }
+                    guard
+                        let appDelegate =
+                            UIApplication.shared.delegate as? AppDelegate,
+                        let scrapViewController =
+                            appDelegate.scrapViewController as? ScrapViewController
+                        else { return }
+                    scrapViewController.scrappedArticles = ScrapManager.fetchArticles()
+                    scrapViewController.setupScrapBadgeValue()
+                    completion(true)
+                    success(true)
+            }
+            if isScrapped {
+                scrapAction.image = UIImage(named: "upload.png")
+                scrapAction.backgroundColor = .red
+            } else {
+                scrapAction.image = UIImage(named: "download.png")
+                scrapAction.backgroundColor = .blue
+            }
+            
+            return scrapAction
+        case .delete:
+            let deleteAction = UIContextualAction(
+                style: .normal,
+                title: nil) { (_, _, success) in
+                    success(true)
+                    completion(true)
+            }
+            deleteAction.image = UIImage(named: "garbage.png")
+            deleteAction.backgroundColor = .red
+            return deleteAction
+        case .markAsRead:
+            let readAction = UIContextualAction(
+                style: .normal,
+                title: nil) { (_, _, success) in
+                    if let articleIdentifier = articleIdentifier,
+                        let articleType: ScrappedArticleType = articleType {
+                        ScrapManager.readArticle(articleType ,articleIdentifier)
```

Comment:
[컨벤션]
정해진 컨벤션에 맞지 않는 띄어쓰기입니다.
```suggestion
                        ScrapManager.readArticle(articleType, articleIdentifier)
```